cfgov/v1/jinja2/v1/_includes/organisms/image-and-text-50-50.html

```diff
@@ -1,20 +1,63 @@
-
 {# ==========================================================================
 
-   image_and_text_50_50.render()
+   image_and_text-50-50.render()
 
    ==========================================================================
 
    Description:
 
-   Create an image and text 50/50 organism.
+   When image conveys critical information and text is supporting it --
+   i.e. chart, excerpt from document
+
    See [GHE]/flapjack/Modules-V1/wiki/50-50-Text-&-Image
 
+   settings:              An object with the following options for settings.
+   settings.title:        A string containing title of module
+   settings.image_path:   A string containing the url for the image
+   settings.image_alt:    A string containing alt text for screenreaders
+   settings.is_widescreen': true,
+   settings.description:  A string for the description under the image
+   settings.is_button:    Boolean if the link should be a button. If false/blank,
+                          the button will remain a plain link.
+   settings.link:         A string for the URL of the link/button
+   settings.link_text:    A string for the text of the link/button call to action
+
    ========================================================================== #}
 
-{% macro render() %}
-<div>
-  {# TODO: Implement image and text 50/50 macro with actual content. #}
-  IMAGE AND TEXT 50/50
+{% macro render(settings) %}
+
+<div class="m-image-text-50-50">
+    {% if settings.image_path %}
```

Comment:
Removed the logic for now in 0661762. I kept it for the link since I could see that being optional in some cases and the organism should still work.